clean_project/project/search_engine.py

```diff
@@ -21,11 +23,50 @@ def search(self, search_word):
         sslmode = load_config_file(self.connect_path).get('sslmode')
         connection = psycopg2.connect(host=host, port=port, sslmode=sslmode,
                                       database=database, user=user, password=password)
+        words = tuple(search_word.split(" "))
```

Comment:
I don't think because it doesn't need of square brackets in the query.(and I remembered that they are into the statement)